src/main/resources/config/application.yml

```diff
@@ -34,6 +34,8 @@ artemis:
     external-system-request:
         batch-size: 50  # wait the time below after 50 requests
         batch-waiting-time: 30000   # in ms = 30s
+    push-notification-relay: https://hermes-sandbox.artemis.cit.tum.de
```

Comment:
The plan that the relay server is hosted by TUM and all Artemis instances use this relay service. That's why I would leave it in the file and other institutions can override it if they want to have their own app instead of using the "official" one.

Reply:
Okay, I’d still like if it was commented out by default if this should become available as a hosted service. This is not only a personal preference but I also think that having this active by default has _legal_ implications:

Right now all our Artemis data is processed inside our own infrastructure. By adding this relay service, TUM becomes a third-party data processor which might be relevant for GDPR and similar data protection regulations. As a responsible admin, I’d like to

- be able to review a privacy statement from your side that tells me exactly which data, where, and how you process it;
- use this information to decide if I feel comfortable activating the service;
- check with University legal/data protection team if it okay to use such a service;
- adapt our privacy policy accordingly.

This all should be an informed and _explicit_ action by the administrator. Therefore, I do not think this should be active by default.